src/data/faq.json

```diff
@@ -466,9 +466,9 @@
-                            "On the one hand, the Robert Koch Institute, which is legally and content-wise responsible for its distribution, is responsible for the delivery of the app. At the moment, the focus is only on delivery via the respective App Stores. Furthermore, the app only works with Google Play Services. With a distribution via the Play Store it can be assumed that Google Play Services are installed. We cannot make these assumptions for other distribution channels at the moment.",
-                            "The distribution of the existing app on other channels is currently published as a request on <a href='https://github.com/corona-warn-app/cwa-wishlist/issues/57' target='_blank' rel='noopener noreferrer'>corona-warn-app/cwa-wishlist#57</a>. The discussion about alternative distribution platforms such as F-Droid can be found at <a href='https://github.com/corona-warn-app/cwa-documentation/issues/5' target='_blank' rel='noopener noreferrer'>corona-warn-app/cwa-documentation#5</a>.",
-                            "Furthermore, there is an initiative in the developer community to make the Corona-Warn-App executable independently of Google Play Services via appropriate extensions. Details can be found in the issue <a href='https://github.com/corona-warn-app/cwa-app-android/issues/75' target='_blank' rel='noopener noreferrer'>corona-warn-app/cwa-app-android#75</a>."
+                            "On the one hand, the Robert Koch Institute, which is legally and content-wise responsible for its distribution, is responsible for the delivery of the app. At the moment, the focus is only on delivery via the respective App Stores.",
+                            "The distribution of the existing app on other channels is currently published as a request on <a href='https://github.com/corona-warn-app/cwa-wishlist/issues/57' target='_blank' rel='noopener noreferrer'>corona-warn-app/cwa-wishlist#57</a>. The discussion about alternative distribution platforms such as F-Droid can be found at <a href='https://github.com/corona-warn-app/cwa-app-android/issues/1483' target='_blank' rel='noopener noreferrer'>corona-warn-app/cwa-app-android#1483</a>.",
+                            "The developer community has developed an inofficial fork of the app under the name <i>Corona Contact Tracing Germany</i> to make the Corona-Warn-App executable independently of Google Play Services via appropriate extensions. However, the fork is not maintained by the Robert Koch-Institut and therefore receives <b>no official support</b>. Details about the development can be found in the issue <a href='https://github.com/corona-warn-app/cwa-app-android/issues/75' target='_blank' rel='noopener noreferrer'>corona-warn-app/cwa-app-android#75</a> as well as in the code repository of the fork: <a href='https://codeberg.org/corona-contact-tracing-germany/cwa-android' target='_blank' rel='noopener noreferrer'>Corona Contact Tracing Germany</a>."

```

Comment:
In connection with your notes in https://github.com/corona-warn-app/cwa-website/issues/971#issuecomment-812393909:

> > Furthermore, there is an initiative in the developer community to make the Corona-Warn-App executable independently of Google Play Services via appropriate extensions. Details can be found in the issue corona-warn-app/cwa-app-android#75.
>
> If this refers to microG, then the initiative has been successful

The initiative that was mentioned in the FAQ was a different project called CoraLibre, which has been [discontinued](https://github.com/CoraLibre/CoraLibre-android#note-on-development-status) and, according to themselves, is "not ready for real-world usage".

The issue corona-warn-app/cwa-app-android#75 does not help explain these circumstances, as it is mostly about CoraLibre development but does not indicate that CoraLibre is a different project than CCTG. For this reason and to avoid such confusion, I recommend against linking this issue here (as it doesn't actually provide "[d]etails about the development" of CCTG).

Reply:
Thank you for the info. I was not aware that CoraLibre was discontinued. That indeed makes linking the issue seem a bit clunky. https://github.com/corona-warn-app/cwa-app-android/issues/75 contains some discussion about CCTG though (although it starts later in the comments) which is why it's linked here. We have to have continuity to the CWA repo. Any external link that is not related to or affiliated with the CWA, RKI, Breg or another state actor will require rigid approval (as @dsarkar already pointed out). Having existing discussions in our repos (and you being a contributor) help in that regard.

Since https://github.com/corona-warn-app/cwa-app-android/issues/75 is technically obsolete, I would recommend you or someone else redoes the issue with updated information on solutions developed by the community. The issue doesn't even have to be about a specific project. Then, we can close the older issue, continue with fresh information and also have something to link to from other issues or the FAQ.

Reply:
@heinezen I opened a new issue that intends to list all current and past CWA-compatible community projects: https://github.com/corona-warn-app/cwa-app-android/issues/2837